Makefile

```diff
@@ -287,6 +300,19 @@ $(CSS_DEST_DIR)/%.css: $(CSS_SRC_DIR)/%.css
 	$(call printfile,$<) >> $@
 
 # maintenance
+ifdef WINDOWS
+clean:
+	$(RM) $(VERSION_CONFIG)
+	$(RM) $(DEFAULTS_CONFIG)
+	$(RM) $(DOCS_PAGE_LIST)
+	$(RM) $(DOCS_VERSION_DATA)
+	$(RM) $(TOC_FILES)
+	$(RM) $(PLUGINS_APP)
+	$(RM) $(FETCHED_FILES)
+	IF EXIST "$(CSS_DEST_DIR)" $(RMDIR) $(CSS_DEST_DIR)
+	IF EXIST "$(PROD_DIR)" $(RMDIR) $(PROD_DIR)
+	IF EXIST "$(DEV_DIR)" $(RMDIR) $(DEV_DIR)
```

Comment:
As above, you can avoid the `if`/`else` here by defining `RM` differently. Also, you can avoid the `IF EXIST` by adding a `-` to the beginning of the line, which will tell Make to ignore any errors. So:
```
-$(RMDIR) $(CSS_DEST_DIR)
```
instead of 
```
$(RMDIR) $(CSS_DEST_DIR)
```